### PR TITLE
Update `rod` deprecated method in the visualizer

### PR DESCRIPTION
**Pull request by flferretti**
This PR updates a deprecated method from `rod` in https://github.com/ami-iit/rod/pull/34 in the visualizer

----
📚 Documentation preview 📚: https://jaxsim--155.org.readthedocs.build//155/

### CODE REVIEW

**Comment by diegoferigo** [comment]
Some of these changes are already part of #148

File: src/jaxsim/mujoco/loaders.py [hunk 1]
```diff
@@ -203,10 +203,8 @@ def convert(
             joints_dict[joint_name].type = "fixed"
 
         # Convert the ROD model to URDF.
-        urdf_string = rod.urdf.exporter.UrdfExporter.sdf_to_urdf_string(
+        urdf_string = rod.urdf.exporter.UrdfExporter.to_urdf_string(
```
Comment (diegoferigo) [inline src/jaxsim/mujoco/loaders.py:206]: Now the options are passed as attributes of the dataclass, can you build the `UrdfExporter` with the same options as before?

**Review by diegoferigo — APPROVED** [review]
Beyond the missing options, LGTM.

File: src/jaxsim/mujoco/loaders.py (hunk 1 above)
Comment (flferretti) [reply src/jaxsim/mujoco/loaders.py]: Thanks, addressed in https://github.com/ami-iit/jaxsim/pull/155/commits/a7bf16d65daa443fb442e62cc191fcbc64115653

**Pull request merged**